### ISSUE TITLE
Test and fix webpack and python doc examples

### ISSUE THREAD

**Issue by jeffbaumes** [Kitware/candela#401]
In particular, the webpack imports should be quoted.

**Comment by ronichoudhury** [comment]
I think instead of mentioning Webpack here, it might make more sense to omit the ES6 example altogether, and put in a small HTML snippet (referencing the CDN version of `candela.js`) in alongside it. That way, if someone copies down the two files, they'll have a working example.

If we do that, the JS would need a change as well (`window.onload` or `$()`).

**Comment by jeffbaumes** [comment]
This is another issue waiting on CDN version of candela that points to the latest release. ⏳
